### PR TITLE
Add ui8/si8 mobilenet benchmarks

### PR DESCRIPTION
**Pull request by jpienaar**
Very early as these are known to not be performant but was thinking of
tracking already.

### CODE REVIEW

File: benchmarks/TFLite/CMakeLists.txt [hunk 1]
```diff
@@ -132,6 +132,32 @@ set(MOBILENET_V2_MODULE
...
+  NAME
+    "MobileNetV2Int8"
+  TAGS
+    "int8,imagenet"
+  SOURCE
+    "https://storage.googleapis.com/tf_model_garden/vision/mobilenet/v2_1.0_int8/mobilenet_v2_1.00_224_int8.tflite"
+  ENTRY_FUNCTION
+    "main"
+  FUNCTION_INPUTS
+    "1x224x224x3xf32"
```
Comment (mariecwhite) [inline benchmarks/TFLite/CMakeLists.txt:158]: Does this INT8 model take in F32 input?

Comment (jpienaar) [reply benchmarks/TFLite/CMakeLists.txt:158]: Surprisingly yes:

```mlir
func.func @main(%arg0: tensor<?x224x224x3xf32> {iree.identifier = "serving_default_input_9:0", tf_saved_model.index_path = ["input_9"]}) -> (tensor<?x1001xf32> {iree.identifier = "StatefulPartitionedCall:0", tf_saved_model.index_path = ["quant_dense_5"]}) attributes {tf_saved_model.exported_names = ["serving_default"]} {
```

At 3 op it gets converted to i8 and then no f32 is used until the 3rd to last it gets converted back to float, so all operations is integral but inputs & output in float.

**Review by GMNGeoffrey — COMMENTED** [review]
Benchmarking is currently pretty slow and resource limited. I'm therefore wary of adding "nice to have" models, especially if they are unoptimized (aka slow) and we are not actively optimizing against them (so it's unclear who's using these numbers)

**Comment by jpienaar** [comment]
> Benchmarking is currently pretty slow and resource limited. I'm therefore easy of adding "nice to have" models, especially if they are unoptimized (aka slow) and we are not actively optimizing against them (so it's unclear who's using these numbers)

That makes sense, these are slow but small models. I completely understand the concern though.

**Comment by GMNGeoffrey** [comment]
Small helps. Can you see empirically how much they slow down the pipeline overall (transferring artifacts is a cost too). Add the "buildkite:benchmark" label to trigger the benchmarks

**Comment by jpienaar** [comment]
(seems buildkite benchmark may be stuck post move)

**Comment by benvanik** [comment]
Closing stale PR.

**Pull request closed without merging**